content/_index.en.md

```diff
@@ -4,8 +4,20 @@ title: Home
 
 # Geo-Distributed Systems
 
-Welcome! This is the landing page for [Rotational Labs'](https://rotational.io/) educational content on the implementation, deployment, and operation of globally distributed computing and storage systems!
+Welcome! This is the landing page for giti educational content on the implementation, deployment, and operation of globally distributed computing and storage systems!
 
 If you'd like to get in touch with us, please feel free to open an issue on GitHub, or reach out to us at info@rotational.io.
 
-Special thanks to [Eucalyp](https://thenounproject.com/eucalyp/) from [The Noun Project](https://thenounproject.com/) who created the logo we're currently using.
\ No newline at end of file
+Special thanks to [Eucalyp](https://thenounproject.com/eucalyp/) from [The Noun Project](https://thenounproject.com/) who created the logo we're currently using.
+
+<!--Rotating globe-->
+<div id="observablehq-canvas-1372f32a"></div>
+
+<script type="module">
+import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
+import define from "https://api.observablehq.com/@kngjoel/untitled.js?v=3";
```

Comment:
Let's please version control this custom JS somewhere here in this repo so that it will not be lost if the observable notebook goes away!

Reply:
Got it!